hardware/firmware/box_rp2040/src/pwr_brd_ctl/fan_ctl.c

```diff
@@ -173,6 +173,13 @@ void pwr_brd_fan_task() {
         uint8_t pwm;
         fan_ctl_get_pwm(i, &pwm);
 
+        if (pwm > FAN_MAX_PWM) {
```

Comment:
We know this is 255, not FAN_MAX_PWM

Reply:
Values from 128 to 255 basically do the same, and values from 11 to 127 are the highest 500-600 RPM, i.e. from 9000 to 9500-9600, I cut them off because the fans spin up just fine with PWM=10, so this is why I have set it to be the maximum value that's ever needed.